index.js

```diff
@@ -182,8 +191,13 @@ ZongJi.prototype.start = function(options) {
           self.connection.pause();
           self._fetchTableInfo(event, function() {
             // merge the column info with metadata
-            event.updateColumnInfo();
-            self.emit('binlog', event);
+            try {
```

Comment:
I think we have a big Try ... catch block somewhere, to catch all the errors during parsing, not necessary here. Yeah, I'd like to have *Try ... catch* as few as possible. (even though, I'm not a Java Hater :wink: )